PD.Workademy.ToDo/src/Web/PD.Workademy.ToDo.Web/Controllers/CategoryController.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using PD.Workademy.ToDo.Web.ApiModels;
+
+namespace PD.Workademy.ToDo.Web.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
```

Comment:
Nema potrebe za ovim posto je vec u ApiBaseController-u

Reply:
to sam izbacio svuda, sta jos treba?